### ISSUE TITLE
Is this repo still maintained ?

### ISSUE THREAD

**Issue by louislebrault** [zenoamaro/react-quill#918]
Hello.

I didn't see any official announcement about it, but last commit is from more than a year ago, and most issues and pull-requests seems ignored.

This surprises me because this package seems still heavily downloaded.

Do the repository owner decided to step back from this project ?

**Comment by alexkrolick** [comment]
Someone asked the [same question](https://github.com/quilljs/quill/issues/3768#issuecomment-1654873923) about Quill. It sounds like they may make a new release at some point, but development is slow.

I might be able to spend some time helping release a new version if/when Quill 2.0 comes out, but other than that I don't have a lot of time to spend on this project, nor do I use it for my day job like I did when I started maintaining it.

Can't speak for Zeno but I assume it's the same situation.

**Comment by Mdkawsarislam2002** [comment]
I don't think so. Last PR is about 1 year ago. And they are not responding in any issue or accepting or rejecting any PR.
